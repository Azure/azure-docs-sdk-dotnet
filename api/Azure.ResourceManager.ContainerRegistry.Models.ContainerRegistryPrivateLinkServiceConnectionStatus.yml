### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus?alt=Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus&text=ContainerRegistryPrivateLinkServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.op_Equality(Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus,Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.op_Inequality(Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus,Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Equals(Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus)
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Disconnected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus
commentId: T:Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus
namespace: Azure.ResourceManager.ContainerRegistry.Models
name: ContainerRegistryPrivateLinkServiceConnectionStatus
fullName: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus
nameWithType: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.ContainerRegistry.dll
packagesWithMoniker:
- value: Azure.ResourceManager.ContainerRegistry v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.ContainerRegistry v1.2.0-beta.1
  monikers:
  - azure-dotnet-preview
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct ContainerRegistryPrivateLinkServiceConnectionStatus : IEquatable<Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type ContainerRegistryPrivateLinkServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure ContainerRegistryPrivateLinkServiceConnectionStatus

      Implements IEquatable(Of ContainerRegistryPrivateLinkServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
summary: The private link service connection status.
metadata:
  api_name:
  - Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.ContainerRegistry.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus
  - Azure::ResourceManager::ContainerRegistry::Models::ContainerRegistryPrivateLinkServiceConnectionStatus
  - ContainerRegistryPrivateLinkServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
