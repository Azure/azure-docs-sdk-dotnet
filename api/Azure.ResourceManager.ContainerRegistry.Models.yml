### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.ContainerRegistry.Models.ArmContainerRegistryModelFactory
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryAgentPoolPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryAgentPoolQueueStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryBaseImageDependency
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryBaseImageTrigger
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryBaseImageTriggerUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryCredentialRegenerateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryCredentials
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryDockerBuildContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryDockerBuildStep
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryDockerBuildStepUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryEncodedTaskRunContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryEncodedTaskStep
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryEncodedTaskStepUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryEncryption
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryFileTaskRunContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryFileTaskStep
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryFileTaskStepUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryGenerateCredentialsContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryGenerateCredentialsResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryImageDescriptor
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryImageUpdateTrigger
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryImportImageContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryImportSource
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryImportSourceCredentials
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryIPRule
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryKeyVaultProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryListCredentialsResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryNameAvailabilityContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryNameAvailableResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryNetworkRuleSet
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryOverrideTaskStepProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPassword
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPlatformProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPlatformUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPolicies
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryReplicationPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryResourceStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryRetentionPolicy
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryRunArgument
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryRunContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryRunGetLogResult
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryRunPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistrySecretObject
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistrySku
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistrySourceTrigger
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistrySourceTriggerDescriptor
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistrySourceTriggerUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTaskOverridableValue
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTaskPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTaskRunContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTaskRunPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTaskStepProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTaskStepUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTimerTrigger
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTimerTriggerDescriptor
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTimerTriggerUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTokenCertificate
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTokenCredentials
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTokenPassword
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTokenPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTriggerProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTriggerUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTrustPolicy
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryUsage
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookCallbackConfig
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookCreateOrUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookEvent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookEventContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookEventInfo
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookEventRequestContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookEventRequestMessage
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookEventResponseMessage
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookEventSource
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookEventTarget
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.CustomRegistryCredentials
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ScopeMapPatch
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.SourceCodeRepoAuthInfo
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.SourceCodeRepoAuthInfoUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.SourceCodeRepoProperties
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.SourceCodeRepoUpdateContent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.SourceUploadDefinition
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
enums:
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPasswordName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.ContainerRegistry.Models.ActionsRequiredForPrivateLinkServiceConsumer
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryBaseImageDependencyType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryBaseImageTriggerType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryCpuVariant
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryEncryptionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryExportPolicyStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryImportMode
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryIPRuleAction
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryNetworkRuleBypassOption
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryNetworkRuleDefaultAction
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryOS
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryOSArchitecture
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPolicyStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryProvisioningState
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPublicNetworkAccess
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryResourceType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryRunStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryRunType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistrySecretObjectType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistrySkuName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistrySkuTier
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistrySourceTriggerEvent
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTaskStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTokenCertificateName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTokenPasswordName
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTokenStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTriggerStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryTrustPolicyType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryUpdateTriggerPayloadType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryUsageUnit
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookAction
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryWebhookStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryZoneRedundancy
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.SourceCodeRepoAuthTokenType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.SourceControlType
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
- uid: Azure.ResourceManager.ContainerRegistry.Models.SourceRegistryLoginMode
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
uid: Azure.ResourceManager.ContainerRegistry.Models
commentId: N:Azure.ResourceManager.ContainerRegistry.Models
name: Azure.ResourceManager.ContainerRegistry.Models
fullName: Azure.ResourceManager.ContainerRegistry.Models
devLangs:
- csharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.ContainerRegistry.Models
  - Azure::ResourceManager::ContainerRegistry::Models
  - Models
  - ContainerRegistry.Models
  - ContainerRegistry::Models
  helpviewer_keywords:
  - Azure.ResourceManager.ContainerRegistry.Models namespace [.NET]
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
