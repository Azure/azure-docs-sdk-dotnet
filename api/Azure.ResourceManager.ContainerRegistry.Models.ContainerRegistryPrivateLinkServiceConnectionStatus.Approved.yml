### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus?alt=Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus&text=ContainerRegistryPrivateLinkServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
  nameWithType: ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As ContainerRegistryPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.ContainerRegistry.Models
name: Approved
fullName: Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
nameWithType: ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.ContainerRegistry.dll
packagesWithMoniker:
- value: Azure.ResourceManager.ContainerRegistry v1.1.0
  monikers:
  - azure-dotnet
- value: Azure.ResourceManager.ContainerRegistry v1.2.0-beta.1
  monikers:
  - azure-dotnet-preview
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
  - Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.ContainerRegistry.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.ContainerRegistry.Models.ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
  - Azure::ResourceManager::ContainerRegistry::Models::ContainerRegistryPrivateLinkServiceConnectionStatus::Approved
  - Approved
  - ContainerRegistryPrivateLinkServiceConnectionStatus.Approved
  - ContainerRegistryPrivateLinkServiceConnectionStatus::Approved
  helpviewer_keywords:
  - ContainerRegistryPrivateLinkServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct ContainerRegistryPrivateLinkServiceConnectionStatus
  monikers:
  - azure-dotnet
  - azure-dotnet-preview
